nano/node/election.hpp

```diff
@@ -69,9 +69,10 @@ class election final : public std::enable_shared_from_this<nano::election>
 	void generate_votes (nano::block_hash const &);
 	void remove_votes (nano::block_hash const &);
 	std::atomic<bool> prioritized_m = { false };
+	bool optimistic_m = { false };
```

Comment:
Thoughts on making this an enum `election_behavior {default, optimistic}` ?

Reply:
Good suggestion